apps/backend/src/mutations/FapMutations.ts

```diff
@@ -229,15 +229,13 @@ export default class FapMutations {
       }
     }
 
-    return this.dataSource
-      .removeMemberFromFap(args, args.roleId === UserRole.FAP_CHAIR)
-      .catch((error) => {
-        return rejection(
-          'Could not remove member from scientific evaluation panel',
-          { agent },
-          error
-        );
-      });
+    return this.dataSource.removeMemberFromFap(args).catch((error) => {
+      return rejection(
+        'Could not remove member from scientific evaluation panel',
```

Comment:
Seems odd that there's still a reference to the SEPs name, but I've opened https://github.com/UserOfficeProject/user-office-project-issue-tracker/issues/1008 for that